src/index.html

```diff
@@ -11,7 +11,153 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <div class="appear">
+        <fieldset>
+          <legend>Personal information</legend>
+          <div class="style">
+            Surname:
+            <input type="text" name="surname" id="surname">
```

Comment:
```suggestion
            <input 
              type="text" 
              name="surname" 
              id="surname"
            >
```

Reply:
Please follow [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) to format your code 
<img width="994" alt="Screenshot 2023-03-15 at 20 55 11" src="https://user-images.githubusercontent.com/32853034/225427702-9b678008-fe4c-4210-a015-3d8cc07ae771.png">

Fix it everywhere